src/components/PlantsV2/components/ObservedNumberOfSpeciesCard.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useState } from 'react';
+import OverviewItemCard from 'src/components/common/OverviewItemCard';
+import strings from 'src/strings';
+import { Box, Typography, useTheme } from '@mui/material';
+import { useAppSelector } from 'src/redux/store';
+import { selectSitePopulation } from 'src/redux/features/tracking/sitePopulationSelector';
+import FormattedNumber from 'src/components/common/FormattedNumber';
+import { ObservationResults } from 'src/types/Observations';
+import { getShortDate } from 'src/utils/dateFormatter';
+import { useLocalization } from 'src/providers';
+
+type ObservedNumberOfSpeciesCardProps = {
+  observation: ObservationResults;
+};
+
+export default function ObservedNumberOfSpeciesCard({ observation }: ObservedNumberOfSpeciesCardProps): JSX.Element {
+  const theme = useTheme();
+  const locale = useLocalization();
+  const populationSelector = useAppSelector((state) => selectSitePopulation(state));
+  const [numReportedSpecies, setNumReportedSpecies] = useState(0);
+  const [numObservedSpecies, setNumObservedSpecies] = useState(0);
+  useEffect(() => {
+    const results = new Set<number>();
+    const species = observation.species.map((s) => s.speciesId).filter((sid) => sid !== undefined);
+    species.forEach((s) => results.add(s!));
+
```

Comment:
The observation species list is already de-duped.
this could be
```
const results = observation.species.filter(s => !!s.id).length;
```
If we want to include uncertain species as well just use the observation.species.length count. There will be species without ids but with a name.

Reply:
Okay, thanks for that feedback. It seems like including uncertain species might be good? I'll put that in and we'll see how that goes.